.golangci.yml

```diff
@@ -0,0 +1,43 @@
+service:
```

Comment:
You might want to use a more aggressive set of linter similar to the ones in https://github.com/mattermost/mattermost-marketplace/pull/27/files#diff-e23b7926f022c85af1665b36d256231b

Just as an idea as these linter have been proven to be valuable.

Reply:
Copied all the linters used